src/convertToObject.js

```diff
@@ -6,7 +6,22 @@
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  const convertedObj = {};
+
+  sourceString.split(';').forEach((elem) => {
+    const startValueIndex = elem.indexOf(':');
+
+    const parametr = elem.slice(0, startValueIndex).trim();
+    const value = elem.slice(startValueIndex + 1).trim();
+
+    parametr.replace(/\n/g, '');
```

Comment:
You are using the `replace` method here to remove newline characters, but you are not reassigning the result back to the `parametr` variable. The `replace` method does not mutate the original string, it returns a new one. So this line of code is not doing anything.